sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverClientTest.java

```diff
@@ -859,4 +863,98 @@ void setSessionState() {
         // Assert
         verify(asyncClient).setSessionState(SESSION_ID, contents);
     }
+
+    @Test
```

Comment:
Could you add a summary with a link to the issue? (Maybe useful to refer back to in the future.) 
 https://github.com/Azure/azure-sdk-for-java/issues/34664

Reply:
Yep, updated the git description.